### ISSUE TITLE
Cannot easily install extra JRE from host

### ISSUE THREAD

**Issue by mickaelistria** [flathub/org.eclipse.Java#5]
I'm using Eclipse SDK Flatpak, and I have multiple versions of Java installed locally (8 and 13 in the host, 11 in Flatpak).
I'd like to configure the Eclipse SDK so that Java 8 and Java 13 are installed (to more easily test backward and forward compatibility of the projects I'm developing).
However, when inside the Installed JRE preference page and trying to add path to those host JREs, I cannot find those path. Then I remember going into `/run/host/usr/lib` but I see the symbolic links can't be selected, so I'm forced to use the exact version, which is likely to become incorrect on next update.

**Comment by marbetschar** [comment]
Would love the possibility to use multiple JDK's with Flatpak Eclipse. Not figured out, how to use a different version than the one bundled and ship with the Flatpak package.

**Comment by mauromol** [comment]
This would be a showstopper for me too, but @mickaelistria correct me if I'm wrong: I can easily select and add the following paths in my Linux Mint 18.3 (based on Ubuntu 16.04):
/run/host/usr/lib/jvm/java-7-openjdk-amd64
/run/host/usr/lib/jvm/java-8-openjdk-amd64
/run/host/usr/lib/jvm/java-11-openjdk-amd64
these are not symlinks, but are paths to generic OpenJDK 7, 8 and 11 versions respectively, and are the paths I would have selected in a normal install too, apart from the fact that they are in /run/host/... in this case. They shouldn't get broken on next OpenJDK update, should they?

BTW: @mbooth101 please document the /run/host/... thing, perhaps it's the solution to my #15 issue...

**Comment by mbooth101** [comment]
> these are not symlinks

I believe @mickaelistria is using Fedora, where they *are* symlinks.

> BTW: @mbooth101 please document the /run/host/... thing, perhaps it's the solution to my #15 issue...

Accessing the host filesystem from sandbox is fully documented at flatpak.org: https://docs.flatpak.org/en/latest/sandbox-permissions.html#filesystem-access

Relevant quote:

> These directories are mounted under /var/run/host: /etc, /usr

**Comment by ojask** [comment]
I've been facing this problem of symbolic links not dereferencing as well. I tried to `Add External JAR `(symlink) into a user library (`Preferences -> Java -> Build Path -> User Libraries`), but it did not work ( `mylib.jar - /path/to/symlink(missing)` ).

Importing the same symlinks through `File -> Import -> FIlesystem` works fine.

I've noticed that the Eclipse uses the the host's filepicker dialog in former and the sandbox's filepicker in the latter.
If this symlink problem can't be fixed, we could have the sandbox's filepicker everywhere for the time being I don't mind going through the `/run/host `everytime.
